commit 9e4fd36af497f07d3b53c701e7dac7b0e1edd734
Author: Li Guanglei <guanglei.li@unisoc.com>
Date:   Thu Nov 26 14:45:04 2020 +0800

    eas: sched_prio_3_rr: extend boundary of pass value
    
    Test run 10s, default timeslice is 100 msecs for SCHED_RR tasks.
    The tolerrance of task opportunity to run is more than 1% when
    floor the calculated value as percentage.
    
    Bug: 174279923
    
    Signed-off-by: Li Guanglei <guanglei.li@unisoc.com>
    Change-Id: I955835f2a4ed004d20d0469aee22ee284f59785a

diff --git a/testcases/kernel/sched/eas/sched_prio_3_rr.c b/testcases/kernel/sched/eas/sched_prio_3_rr.c
index 8403da5fe..b4ecbbbaf 100644
--- a/testcases/kernel/sched/eas/sched_prio_3_rr.c
+++ b/testcases/kernel/sched/eas/sched_prio_3_rr.c
@@ -24,7 +24,7 @@
 
 #define TRACE_EVENTS "sched_wakeup sched_switch sched_process_exit"
 
-#define EXEC_MIN_PCT 33
+#define EXEC_MIN_PCT 32
 #define EXEC_MAX_PCT 34
 
 static sem_t sem;
